src/js/InstitutionStatus.jsx

```diff
@@ -52,6 +52,7 @@ var InstitutionStatus = React.createClass({
         break;
       case 9:
       case 10:
+        statusLink = <Link to={appLink}>Sign your filing</Link>
```

Comment:
Don't we still want this link here? Since we still need to open the submission container view to do the actual signing.